config/.metrics_rubocop.yml

```diff
@@ -0,0 +1,93 @@
+# Metrics::AbcSize
+# (Assignment Branch Condition)
+#
+# Checks that the ABC size of methods is not higher thatn the configured maximum
+
+Metrics/AbcSize:
+  Max: 15
+
+# Metrics::BlockLength
+#
+# Checks if the length of a block exceeds some maximum value.
+# Comment lines are ignored
+
+Metrics/BlockLength:
+  CountComments: false
+  Max: 25
+  Exclude:
+    - 'Rakefile'
+    - '**/*.rake'
+    - 'spec/**/*.rb'
+
+# Metrics::BlockNesting
+#
+# Checks for excessive nesting of conditional and looping constructs
+
+Metrics/BlockNesting:
+  Max: 3
+
+# Metrics::ClassLength
+#
+# Checks if the length of a class exceeds some maximum value.
+# This can be used to ensure 'one responsibility per class" rule.
+
+Metrics/ClassLength:
+  CountComments: false
+  Max: 100
```

Comment:
I am not sure how realistic this is with regard to the platform, e.g. the `product` model is 416 lines long.